zmsclient/src/Zmsclient/OidcHandler.php

```
@@ -0,0 +1,142 @@
+
+    /**
+     * @return array{workstation: mixed, department_count: int, redirect_to_index: bool}
+     */
+    private function authenticateWorkstation(string $application, string $sessionHash): array
+    {
+        try {
+            $workstation = $this->http
+                ->readGetResult('/workstation/', ['resolveReferences' => 2])
+                ->getEntity();
+
+            $username = $workstation->getUseraccount()->id;
```

PHPStan (2.1.54)

[error] Cannot call method getUseraccount() on BO\Zmsentities\Schema\Entity|false.


(method.nonObject)